packages/batch-tx-handler/package.json

```
@@ -34,6 +35,7 @@
     "dotenv-expand": "^10.0.0",
     "ethereumjs-util": "^7.1.5",
     "ethers": "^5.7.0",
+    "node-fetch": "2",
```

this does not look like a proper semver
why is it not the latest ver 3.3.2?
https://www.npmjs.com/package/node-fetch

---

This is current a CommonJS project, we may change it to esm at another pr then upgrade the version?

---

if you want to set to a specific version, it should still follow the standard semver format, for example the latest of v2 is [2.7.0](https://www.npmjs.com/package/node-fetch/v/2.7.0)

so i'd put 
```suggestion
    "node-fetch": "2.7.0",
```

or according to your `yarn.lock` it's 2.6.7, so put 2.6.7

---

Oh, I see, ty